modules/local/starfusion/build/main.nf

```diff
@@ -20,11 +21,21 @@ process STARFUSION_BUILD {
     script:
     def args = task.ext.args ?: ''
     """
+    export TMPDIR=/tmp
```

Comment:
This will cause crashes on our infrastructure, any chance we can use ./tmp instead?

Reply:
Yes! Thank you for reviewing! I'll change this.